build.sbt

```diff
@@ -54,6 +54,15 @@ libraryDependencies ++= Seq(
   "io.github.classgraph" % "classgraph" % "4.8.53" % Test
 )
 
+// Benchmarking
+
+enablePlugins(JmhPlugin)
+sourceDirectory in Jmh := (sourceDirectory in Test).value
+classDirectory in Jmh := (classDirectory in Test).value
+dependencyClasspath in Jmh := (dependencyClasspath in Test).value
+compile in Jmh := (compile in Jmh).dependsOn(compile in Test).value
+run in Jmh := (run in Jmh).dependsOn(compile in Jmh).evaluated
```

Comment:
I think it's better to put the benchmarks into a separated module

And then the JMH plugin could only be activated for the benchmarks.

Ex: https://github.com/circe/circe/blob/6d2e2dddc42c7b38f5ed35527d5caad8b33bff35/build.sbt#L476-L487

Reply:
Makes sense, I will have a look at this.